EssaGUI/gui/HostWindow.cpp

```diff
@@ -92,7 +91,7 @@ void HostWindow::handle_events() {
 void HostWindow::do_draw() {
     // hacky hacky hacky hacky
     window().clear();
-    llgl::opengl::clear(llgl::opengl::ClearMask::Depth);
+    glClear(GL_DEPTH_BUFFER_BIT);
```

Comment:
It could still be dedicated function for clearing buffer

Reply:
yes